app/javascript/components/util/DateInput.js

```diff
@@ -116,6 +126,7 @@ DateInput.propTypes = {
   placement: PropTypes.oneOf(['top', 'bottom', 'left', 'right', 'auto']),
   isInvalid: PropTypes.bool,
   isClearable: PropTypes.bool,
+  requireDate: PropTypes.bool,
```

Comment:
a bit nitpicky, but can we just call this `required`

Reply:
yup changed here: https://github.com/SaraAlert/SaraAlert/pull/734/commits/ec749532c1eb08e4f15e744e940896afc4f87bea